packages/demo-app/src/examples/IconExample.tsx

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2021 Palantir Technologies, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.*/
+
+import * as React from "react";
+
+import { Icon, Intent } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+
+import { ExampleCard } from "./ExampleCard";
+
+export class IconExample extends React.PureComponent {
+    public render() {
+        return (
+            <div className="example-row">
+                <ExampleCard horizontal={true}>
+                    {Object.values(Intent).map(intent => (
```

Comment:
I think this section needs a label of some kind... it took me a second to realize which component was being demo'd

![image](https://user-images.githubusercontent.com/723999/134342067-bfc53e08-479f-41b2-b263-8d7f0190b1b1.png)